build.gradle

```diff
@@ -18,7 +18,7 @@ buildscript {
     ext {
         // When upgrading to version after 7.3.0, must remove also
         // project substitution from configurations.all (line 39)
-        es_version = System.getProperty("es.version", "7.3.0")
+        es_version = System.getProperty("es.version", "7.3.2")
```

Comment:
Please change the 2 plugin versions accordingly. Reference: https://github.com/opendistro-for-elasticsearch/sql/pull/214/files